src/gc/gc.cpp

```diff
@@ -1973,7 +1973,7 @@ BOOL Aligned (size_t n)
 #define OBJECT_ALIGNMENT_OFFSET (sizeof(MethodTable *))
 
 #ifdef FEATURE_STRUCTALIGN
-#define MAX_STRUCTALIGN OS_PAGE_SIZE
+#define MAX_STRUCTALIGN GC_PAGE_SIZE
```

Comment:
max alignment supported for this feature is supposed to be an OS page size (so you could align objects per page if you want to). there's not really a "GC page size" even though there's now a name GC_PAGE_SIZE (which is exactly a good name, IMO. it makes it sound like GC has some kind of expectation of what a page size should be).